better-mouse-trap/README.md

```diff
@@ -2,6 +2,103 @@
 
 A smarter mouse trap with SMS alerting.
 
-![](../app-banners/nf3.png)
+Find the complete story in the detailed [Hackster.io writeup].
 
-You can find all instructions for building this project in [its Hackster writeup](https://www.hackster.io/hendersoncarlton/i-love-checking-on-mousetraps-said-no-one-ever-52c5e7). This repo contains the project’s [JSONata configuration](./mousetrap.jsonata) for reference.
\ No newline at end of file
+[![Mouse wearing a helmet grabbing cheese from a trap](../app-banners/nf3.png)][hackster.io writeup]
+
```

Comment:
As a suggestion I like having a mini TOC at the top of these guides. For example see the "To run this project yourself” sections at the top of https://github.com/blues/app-accelerators/tree/main/indoor-floor-level-tracker and https://github.com/blues/app-accelerators/tree/main/fridge-fleet-monitor.